README.md

```diff
@@ -32,12 +32,21 @@ For running locally in container run
 
 1. `docker-compose -f docker-compose.yml build`
 2. `docker-compose -f docker-compose.yml up`
-3. `docker-compose -f docker-compose.metrics.yml up`
-
-To configure grafana go to `http://localhost:8000/dashboards` and dashboards from `./grafana` folder.
 
 For running KAPI, one can also use the image from this page https://docs.lido.fi/guides/tooling#keys-api. Please always use the SHA256 hash of the Docker image for the latest release: lidofinance/lido-keys-api@<latest-hash>.
 
+## Metrics and alerts
+
+To launch Prometheus, Grafana, and Alertmanager, execute the following command:
+
+```
+docker-compose -f docker-compose.metrics.yml up
+```
+
+For configuring Grafana, navigate to http://localhost:8000/dashboards. Here, you can import dashboards from the ./grafana folder.
```

Comment:
To configure Grafana dashboards, navigate to `URL` and import dashboards from the `./grafana` folder.
Export them after you done?

Reply:
i think import is enough, we dont update outr grafana based on this dashboards
we do visa versa

Reply:
and use this dashboards for local development